app/models/solarcycle.rb

```diff
@@ -1,4 +1,6 @@
 class Solarcycle < ApplicationRecord
   self.inheritance_column = :_type_disabled
   belongs_to :location
+
+  scope :after_beginning_of_yesterday, -> { where('created_at < ?', 1.day.ago.beginning_of_day) }
```

Comment:
Just for clarification, is this to limit checking solarcycles after yesterday, or to only getting the solarcycles from the day before to limit the amount in the db?

Reply:
The latter. Otherwise we'll always be sending all solarcycles in the past and the network response will become slow.